typescript/web/src/components/import-button/import-images-modal/types.ts

```diff
@@ -24,7 +24,14 @@ export type DroppedUrl = {
  */
 export type UploadStatuses = Record<string, boolean | string>;
 
+export type UploadInfo = {
+  status: boolean | string;
```

Comment:
Why do you need to use boolean | string? Isn't `?: string` sufficient?

Reply:
it has 3 states: not uploaded (false), uploaded (true), or error (string).

Reply:
OK thanks for the answer :) Then can we do a **_quick-change_** to use this instead?

```ts
status: FileUploadStatus
error?: string;
```

```ts
type FileUploadStatus = "pending" | "uploading" | "uploaded" | "error";
```

**_Side-note:_** _If we started from scratch, we could have used a [react-query](https://react-query.tanstack.com/overview) mutation which mimicks the Apollo `useMutation` output ( with the `data`, `error` and `loading` names etc.), hence simplifying and standardizing the interface we're talking about 🚀_  
_But changes would have been too big for this PR anyway, it's just some knowledge-sharing 😉 You've done the right job!_